app/src/main/java/uk/org/openseizuredetector/MainActivity.java

```diff
@@ -963,7 +963,7 @@ public void run() {
                     tv.setBackgroundColor(warnColour);
                     tv.setTextColor(warnTextColour);
                     tv = (TextView) findViewById(R.id.data_time_tv);
-                    tv.setText(mConnection.mSdServer.mSdData.dataTime.format("%H:%M:%S"));
+                    tv.setText(Calendar.getInstance().getTime().toString());
```

Comment:
You are changing the displayed date/time from the data time to the current time - I wonder why?   I find using the data time useful to check that data is arriving at the expected frequency.